### PR TITLE
fix wrong url

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,8 @@ datasources:
     basicAuthPassword: test_password
 ```
 
+Above setting is useful to run loki locally, if use `docker` or `docker-compose`, need change url as `http://loki:3100`.
```
Comment (davkal) [inline README.md]: I'd rephrase this to be more general:

Make sure to adjust the `url` and authentication to your needs, e.g., in the `docker-compose` example of the Loki URL is `http://loki:3100` and no authentication is needed.

**Comment by daixiang0** [comment]
I has updated it

**Comment by davkal** [comment]
Thank you for your contribution @daixiang0
